### PR TITLE
OHRM-1040:Support page improvements

### PR DESCRIPTION
**Pull request by KaveeshaSilva1996**

### CODE REVIEW

File: symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php [hunk 1]
```diff
@@ -114,10 +138,19 @@ protected function sendQuery($url, $contentType = self::DEFAULT_CONTENT_TYPE) {
         );
     }
 
+    /**
+     * @return string
+     */
     public function getDefaultRedirectUrl() {
-        return $this->getBaseUrl().'/hc/en-us';
+        return $this->getBaseUrl().self::ZENDESK_DEFAULT_REDIRECT_DIRECTORY;
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php:145]: Not directory right? `ZENDESK_DEFAULT_URL_PATH`

File: symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php [hunk 2]
```diff
@@ -22,6 +22,7 @@ class ZendeskHelpProcessor implements HelpProcessor {
     const DEFAULT_CONTENT_TYPE = "application/json";
     const ZENDESK_SEARCH_DIRECTORY_URL = '/api/v2/help_center/articles/search.json?';
     const ZENDESK_CATEGORY_DIRECTORY_URL = '/api/v2/help_center/categories';
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php:24]: Why we defined const as `DIRECTORY` those are API endpoints :thinking:

File: symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php [hunk 3]
```diff
@@ -22,6 +22,7 @@ class ZendeskHelpProcessor implements HelpProcessor {
     const DEFAULT_CONTENT_TYPE = "application/json";
     const ZENDESK_SEARCH_DIRECTORY_URL = '/api/v2/help_center/articles/search.json?';
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php:23]: Is this `.json` required. Also, you can move ? to particular place, rather than here

File: symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php [hunk 4]
```diff
@@ -41,10 +42,19 @@ public function setHelpConfigService($helpConfigService) {
...
     }
 
+    /**
+     * @param null $query
+     * @param array $labels
+     * @param array $categorieIds
+     * @return false|string
+     */
     public function getSearchUrlFromQuery($query=null,$labels=[],$categorieIds=[]) {
         $mainUrl=$this->getBaseUrl().self::ZENDESK_SEARCH_DIRECTORY_URL;
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php:59]: Better to format these codes though IDE

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php (hunk 3 above)
Comment (KaveeshaSilva1996) [reply symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php:23]: removed the DIRECTORY part of the variable

File: symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php (hunk 2 above)
Comment (KaveeshaSilva1996) [reply symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php:24]: removed the DIRECTORY part of the variable

**Review by RajithaKumara — CHANGES_REQUESTED** [review]
Travis CI failing, can you check @KaveeshaSilva1996

File: dbscript/dbscript-2.sql [hunk 5]
```diff
@@ -3536,3 +3536,26 @@ INSERT INTO `hs_hr_config` (`key` ,`value`) VALUES ('help.url',  'https://openso
 INSERT INTO `hs_hr_config` (`key` ,`value`) VALUES ('help.processorClass',  'ZendeskHelpProcessor');
 INSERT INTO `ohrm_i18n_group` (`name`,`title`) VALUES ('help','Help');
 
+INSERT INTO `ohrm_screen` (`name`, `module_id`, `action_url`) VALUES
```
Comment (RajithaKumara) [inline dbscript/dbscript-2.sql:3539]: Can you revert these changes?

File: upgrader/apps/upgrader/lib/schemaIncrementTasks/SchemaIncrementTask80.php [hunk 6]
```diff
@@ -35,6 +35,28 @@ public function loadSql()
 
         $sql[] = "INSERT INTO `ohrm_i18n_group` (`name`,`title`) VALUES ('help','Help')";
 
+        $sql[] =  "INSERT INTO `ohrm_screen` (`name`, `module_id`, `action_url`) VALUES
```
Comment (RajithaKumara) [inline upgrader/apps/upgrader/lib/schemaIncrementTasks/SchemaIncrementTask80.php:38]: Can you revert these changes as well?

**Review by RajithaKumara — CHANGES_REQUESTED** [review]
